addons/web/static/src/views/basic_relational_model.js

```diff
@@ -243,6 +243,11 @@ export class Record extends DataPoint {
                 case "integer":
                 case "monetary":
                     continue;
+                case "html":
+                    if (this.isRequired(fieldName) && this.data[fieldName] && this.data[fieldName].length === 0) {
```

Comment:
isn't `this.data[fieldName]` always a markup? I think you can get rid of the check